### DISCUSSION TITLE
My wifi/internet breaks every time I use nuclei scanner.

### DISCUSSION THREAD

**Discussion by Ammitte** [projectdiscovery/nuclei-templates#8723] [Q&A]
Why my wifi/internet stops working every time on every device when I use nuclei scanner and when I stop scanning my internet starts working.Im using nuclei on Kali Linux 2023.3 and I'm facing this issue on every nuclei scan.

**Comment by MetzinAround** [comment]
Hi Ammite, unfortunately this is likely something todo with your ISP provider noticing strange traffic from your IP Address while scanning. IT's recommended that you scan from a different IP than your home IP. https://medium.com/coinmonks/dont-get-arrested-should-you-use-vpn-for-bug-bounty-hunting-c39019f34f10#:~:text=The%20Bug%20Bounty%20community%20has,Virtual%20Private%20Network%20(VPN).
